### PR TITLE
Increase Starknet Bridge limit to 5m

### PR DESCRIPTION
**Pull request by maciejka**

### CODE REVIEW

**Comment by 0xdecr1pto** [comment]
@maciejka Can you please fix merge conflicts buy reverting removing of whitespaces

**Comment by maciejka** [comment]
@0xdecr1pto conflicts resolved

**Review by amusingaxl — APPROVED** [review]
Let's merge this into the main spell to avoid merge conflicts.

File: src/DssSpell.sol [hunk 1]
```diff
@@ -66,6 +66,10 @@ interface RwaInputConduitLike {
     function file(bytes32 what, address data) external;
 }
 
+interface StarknetLike {
+    function setCeiling(uint256) external;
```
Comment (The-Arbiter) [inline src/DssSpell.sol:70]:
```suggestion
    function setCeiling(uint256 _ceiling) external;
```

Interface needs updating to match on-chain function spec

File: src/DssSpell.sol [hunk 2]
```diff
@@ -248,6 +254,11 @@ contract DssSpellAction is DssAction {
         GemAbstract(RWA014).approve(RWA014_A_URN, 1 * WAD);
         RwaUrnLike(RWA014_A_URN).lock(1 * WAD);
 
+        // ---------- Starknet ----------
+        // Increase Starknet Bridge Limit from 1,000,000 DAI to 5,000,000 DAI
+        // https://forum.makerdao.com/t/april-26th-2023-spell-starknet-bridge-limit/20589
```
Comment (The-Arbiter) [inline src/DssSpell.sol:259]:
```suggestion
        // Forum: https://forum.makerdao.com/t/april-26th-2023-spell-starknet-bridge-limit/20589
```

**Review by The-Arbiter — APPROVED** [review]
Confirmed precision is WAD
Some style-based nits but LGTM overall

**Comment by amusingaxl** [comment]
@maciejka New merge conflicts. Can you please fix them so we can merge and leave you alone 😅 ?

**Pull request merged**